### ISSUE TITLE
Error on 21. Migration

### ISSUE THREAD

**Issue by msalihkarakasli** [anchorcms/anchor-cms#1178]
### Summary

On first access of web page after installation, 21. migration throw an exception.
"**Null value not allowed: 1138 Invalid use of NULL value**"

### Expected Behaviour

- All migrations should be ran without error.

### Actual Behaviour

- 21. migration "21_alter_comment_date.php" doesn't works properly.

### Context details (if applicable)

- Anchor version: 0.12.1
- Server setup:
- URL

**Comment by TheBrenny** [comment]
I did a fresh install three days ago and didn't hit the problem...

What's your server setup?

**Comment by msalihkarakasli** [comment]
Server: 5.7.17-0ubuntu0.16.04.1
PHP Version: 7.0.13-0ubuntu0.16.04.1
Database: libmysql - mysqlnd 5.0.12-dev

AFTER keyword does not support for CHANGE keyword. When i remove AFTER part. It was ok and worked.

**Comment by TheBrenny** [comment]
Fair enough. I'm glad to hear that it worked for you in the end!

What's happening is that MySQL gives the option to reorder the columns, which is what that `AFTER status` part is. Don't know why it didn't work for you though... 🤔

**Comment by CraigChilds94** [comment]
@msalihkarakasli What is your version of MySQL?

**Comment by frankn11** [comment]
Got the same problem.
PHP 7.0.14 (cli)
mysql  Ver 14.14 Distrib 5.7.17, for macos10.12 (x86_64)

You only removed the AFTER part in the migration and it worked?

**Comment by msalihkarakasli** [comment]
Right. I also got the same problem for same migrations 21 and 61. 

This is enough;
ALTER TABLE ' . $table . ' CHANGE date date datetime NOT NULL
